client/src/components/page/StatsPage.js

```diff
@@ -40,10 +40,11 @@ class StatsPage extends Component {
 
   render() {
     return (
-      <p>
-        The total number of messages:
-        {this.state.messageCount}
-      </p>
+      <div id='content'>
```

Comment:
Let's not use id's because usually an id indicates that you want to reference this specific div from code. Usually in react, we don't need to use ids because we write all of our HTML nested in our javascript code. Since we don't need a reference to this div, I would just leave the div as <div>